src/test/java/ua/ithillel/lesson4/ArrayValueCalculatorTest.java

```diff
@@ -0,0 +1,51 @@
+package ua.ithillel.lesson4;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ArrayValueCalculatorTest {
+    @Test
+    void test1() throws ArrayDataException {
+        ArrayValueCalculator array = new ArrayValueCalculator();
```

Comment:
```suggestion
        ArrayValueCalculator calculator = new ArrayValueCalculator();
```
це ж не array, а калькулятор, тому calculator було б правильніше